### PR TITLE
Saxy.stream_events implemented

### PR DESCRIPTION
**Pull request by hissssst**
Dead simple accumulating handler added
ExDoc bumped

### CODE REVIEW

**Comment by hissssst** [comment]
@qcam hi, thanks for the library, I've implemented feature which transforms XML binary stream into events stream

**Comment by hissssst** [comment]
@qcam test pipeline is silently failing

**Comment by qcam** [comment]
@hissssst Could you please rebase to master? The CI failure should be fixed now.

File: lib/saxy.ex [hunk 1]
```diff
@@ -309,10 +312,95 @@ defmodule Saxy do
...
+
+  ### Options
+
+  See the “Shared options” section at the module documentation.
+
+  * `:character_data_max_length` - tells the parser to emit the `:characters` event when its length exceeds the specified
+    number. The option is useful when the tag being parsed containing a very large chunk of data. Defaults to `:infinity`.
+  """
+  @spec map_stream!(in_stream :: Enumerable.t(), options :: Keyword.t()) :: out_stream :: Enumerable.t()
+  def map_stream!(stream, options \\ []) do
```
[the remark this answers is not in the document]
Comment (hissssst) [reply lib/saxy.ex:365]: You can rename it in any way yourself, I have no problem with `stream_events`

**Comment by hissssst** [comment]
Done

**Comment by qcam** [comment]
Thank you!

**Pull request merged**